toolkit-distribution/src/release/LICENSE

```diff
@@ -533,3 +538,24 @@ For details, see http://asm.objectweb.org/asm/
 You can find a copy of the License at licenses/LICENSE-asm
 
 * ASM Core (org.ow2.asm:asm:5.0.4 - http://asm.objectweb.org/asm/)
+
+===========================================================================
+This product bundles jmustache which is licensed under The (New) BSD License.
+For details, see http://github.com/samskivert/jmustache
+You can find a copy of the License at licenses/LICENSE-jmustache
+
+* jmustache (com.samskivert:jmustache:1.14 - http://github.com/samskivert/jmustache)
+
+===========================================================================
+This product bundles JSch which is licensed under Revised BSD.
+For details, see http://www.jcraft.com/jsch/
+You can find a copy of the License at licenses/LICENSE-jsch
+
+* JSch (com.jcraft:jsch:0.1.54 - http://www.jcraft.com/jsch/)
+
+===========================================================================
+This product bundles JZlib which is licensed under Revised BSD.
+For details, see http://www.jcraft.com/jzlib/
```

Comment:
should be http://www.jcraft.com/jzlib/LICENSE.txt

Reply:
I have correct the URL in the new commit